fix dependency in Makefiles

there is no longer a sysdep.h file.

Signed-off-by: Francois Perrad <francois.perrad@gadz.org>

diff --git a/Makefile b/Makefile
index 976d00a..7f73c44 100644
--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,7 @@ libficl.so.$(MAJOR).$(MINOR): $(OBJECTS)
 	-o libficl.so.$(MAJOR).$(MINOR) $(OBJECTS)
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so
 
-main: main.o ficl.h sysdep.h libficl.so.$(MAJOR).$(MINOR)
+main: main.o ficl.h libficl.so.$(MAJOR).$(MINOR)
 	$(CC) $(CFLAGS) $(LDFLAGS) main.o -o main -L. -lficl -lm
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so.$(MAJOR)
 
diff --git a/Makefile.ansi b/Makefile.ansi
index e510fcd..170e6eb 100644
--- a/Makefile.ansi
+++ b/Makefile.ansi
@@ -29,7 +29,7 @@ libficl.so.$(MAJOR).$(MINOR): $(OBJECTS)
 	-o libficl.so.$(MAJOR).$(MINOR) $(OBJECTS)
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so
 
-main: main.o ficl.h sysdep.h libficl.so.$(MAJOR).$(MINOR)
+main: main.o ficl.h libficl.so.$(MAJOR).$(MINOR)
 	$(CC) main.o -o main -L. -lficl -lm
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so.$(MAJOR)
 
diff --git a/Makefile.linux b/Makefile.linux
index d447e7e..7f2cdfe 100644
--- a/Makefile.linux
+++ b/Makefile.linux
@@ -29,7 +29,7 @@ libficl.so.$(MAJOR).$(MINOR): $(OBJECTS)
 	-o libficl.so.$(MAJOR).$(MINOR) $(OBJECTS)
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so
 
-main: main.o ficl.h sysdep.h libficl.so.$(MAJOR).$(MINOR)
+main: main.o ficl.h libficl.so.$(MAJOR).$(MINOR)
 	$(CC) main.o -o main -L. -lficl -lm
 	ln -sf libficl.so.$(MAJOR).$(MINOR) libficl.so.$(MAJOR)
 
-- 

